controllers/movies.js

```diff
@@ -0,0 +1,55 @@
+const models = require('../models')
+
+const getAllProjects = async (request, response) => {
+  try {
+    const allMovies = await models.Movies.findAll()
+    const allSeries = await models.Series.findAll()
+    const allProjects = {
+      movies: allMovies,
+      series: allSeries
+    }
+
+    return (allProjects.movies.length > 0 || allProjects.series.length > 0)
+      ? response.send(allProjects)
+      : response.status(404).send('No projects found, please try again.')
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve projects, please try again.')
+  }
+}
+
+const getMoviesByName = async (request, response) => {
+  try {
+    const { name } = request.params
+
+    const movie = await models.Movies.findAll({
+      where: {
+        name: { [models.Op.like]: `%${name}%` },
+      },
+      include: [{
+        model: models.Actors
+      }]
+    })
+
+    return movie.length > 0
+      ? response.send(movie)
+      : response.status(404).send('No movies found by that name, please try again.')
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve movies, please try again.')
+  }
+}
+
+const saveNewMovie = async (request, response) => {
+  try {
+    const { name, yearReleased, director } = request.body
```

Comment:
Your docs say `{ "movie": "New Movie", releaseDate": "year", "director": "Director" }` so this route doesn't work as expected